[AIDemos] Add missing operators for AIDemos

xta0 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#36756 [AIDemos] Add missing operators for AIDemos**

1. Add missing ops for pytorch models used in AIDemos. This is because we're migrating towards the lite-interpreter on mobile, the full JIT version will be deprecated
2. Replace the old mobilenet model with a newer one in bytecode format
3. Regenerate the reddit model to include bytecode

Differential Revision: [D21013409](https://our.internmc.facebook.com/intern/diff/D21013409/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D21013409/)!

xta0 committed:
[AIDemos] Add missing operators for AIDemos

1. Add missing ops for pytorch models used in AIDemos. This is because we're migrating towards the lite-interpreter on mobile, the full JIT version will be deprecated
2. Replace the old mobilenet model with a newer one in bytecode format
3. Regenerate the reddit model to include bytecode

Differential Revision: [D21013409](https://our.internmc.facebook.com/intern/diff/D21013409/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D21013409/)!

[ghstack-poisoned]

eellison commented:
Why are we moving ops back and forth from register_prim_ops to register_prim_ops_fulljit.cpp.cpp  ? (in the initial move and then moving them back now ?)

xta0 committed:
Update on "[AIDemos] Add missing operators for AIDemos"


1. Add missing ops for pytorch models used in AIDemos. This is because we're migrating towards the lite-interpreter on mobile, the full JIT version will be deprecated
2. Replace the old mobilenet model with a newer one in bytecode format
3. Regenerate the reddit model to include bytecode

Differential Revision: [D21013409](https://our.internmc.facebook.com/intern/diff/D21013409/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D21013409/)!

[ghstack-poisoned]

xta0 committed:
Update on "[AIDemos] Add missing operators for AIDemos"


1. Add missing ops for pytorch models used in AIDemos. This is because we're migrating towards the lite-interpreter on mobile, the full JIT version will be deprecated
2. Replace the old mobilenet model with a newer one in bytecode format
3. Regenerate the reddit model to include bytecode

Differential Revision: [D21013409](https://our.internmc.facebook.com/intern/diff/D21013409/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D21013409/)!

[ghstack-poisoned]
